saleor/graphql/order/mutations/order_grant_refund_create.py

```diff
@@ -183,7 +183,7 @@ def clean_input(
         input: dict[str, Any],
     ):
         amount = input.get("amount")
-        reason = input.get("reason", "")
+        reason = input.get("reason") or ""
```

Comment:
When it makes a difference? For me it looks like is the same

Reply:
`reason` is nullable.
You can get `input == {"reason": None}`, and `.get` with `default=""` will return `None` as the key exist.